lib/partition_healing/discover_provider_healer.js

```diff
@@ -63,6 +63,7 @@ require('util').inherits(DiscoverProviderHealer, Healer);
  */
 DiscoverProviderHealer.prototype.heal = function heal(callback) {
     var self = this;
+    self.ringpop.stat('increment', 'heal.triggered.discover_provider');
```

Comment:
the stat we emit here differs from the stat in `ringpop-go`: https://github.com/uber/ringpop-go/blob/dev/ringpop.go#L497 is that on purpose?